### PR TITLE
Add tappable ? to show help and remove NaN

### PR DESCRIPTION
**Pull request by markulrich**

### CODE REVIEW

**Comment by markulrich** [comment]
image

**Comment by tinaroh** [comment]
not a huge deal but why is it an oval? haha

File: src/pages/Metrics.tsx [hunk 1]
```diff
@@ -300,43 +301,64 @@ function getRetention(
...
+        const change = curr / last - 1.0;
+        const changeStyle = {color: change > 0 ? 'green' : (change < 0 ? 'red' : 'gray')};
+        const [lastDis, currDisp, changeDisp]  = [last, curr, change].map((x, i) => isNaN(x) ? '--' : (i === 2 || curr_m ? `${(x * 100).toFixed(2)}%` : x));
+        const { name, desc } = metricTemplate;
+        const key = `${durationDays}D${name}M`;
+        const showDesc = this.state[key];
+        metricFrags.push(
+          <div style={styles.metric} title={desc} key={key}>
+            {showDesc && <div>{desc}</div>}
+            {currDisp}{curr_m ? ` ${curr_m} ` : ' '} {name}, <span style={changeStyle}>{`${change > 0 ? '+' : ''}${changeDisp}`}</span> from {lastDis} <span onClick={() => this.setState(s => ({[key]: !s[key]}))} style={styles.circle}>{showDesc ? '-' : '?'}</span>
```
Comment (tinaroh) [inline src/pages/Metrics.tsx]: Can you hide the entire circle when `showDesc` is false?

**Comment by tinaroh** [comment]
It's not that internal, we're linking it in the newsletter

**Comment by markulrich** [comment]
image

**Pull request merged**